apps/devhub/tests/test_views_versions.py

```diff
@@ -186,6 +187,24 @@ def test_user_can_disable_addon(self, hide_mock):
         msg = entry.to_string()
         assert self.addon.name.__unicode__() in msg, ("Unexpected: %r" % msg)
 
+    @mock.patch('devhub.views.unindex_addons')
+    def test_user_can_unlist_addon(self, unindex):
+        self.addon.update(status=amo.STATUS_PUBLIC, disabled_by_user=False,
+                          is_listed=True)
+        res = self.client.post(self.unlist_url)
+        self.assertEqual(res.status_code, 302)
```

Comment:
You've modified the rest of the test, but it seems you missed this line:
`assert res.status_code == 302`